examples/ipc_level_zero/utils_level_zero.h

```diff
@@ -0,0 +1,224 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#include <stdlib.h>
+
+#include <level_zero/ze_api.h>
+
+static int init_level_zero() {
```

Comment:
@bratpiorka I think these utility functions can be re-used in the Level Zero provider unit tests. But we can do it in a separate PR.